capsules/src/lsm303dlhc.rs

```diff
@@ -0,0 +1,779 @@
+//! Driver for the LSM303DLHC 3D accelerometer and 3D magnetometer sensor.
+//!
+//! May be used with NineDof and Temperature
+//!
+//! I2C Interface
+//!
+//! <https://www.st.com/en/mems-and-sensors/lsm303dlhc.html>
+//!
+//! Syscall Interface
+//! -----------------
```

Comment:
I might suggest removing this here and putting a link to the documentation instead. I'm always wary when things are documented in two places, easy to accidentally get out of sync, and which is the authority?

Reply:
Are you suggesting to remove the syscall interface and place a link towards the markdown file?

Reply:
Yep!

Reply:
Done